tensorboard/plugins/profile/profile_demo.py

```diff
@@ -33,6 +33,7 @@
 from tensorboard.plugins.profile import profile_demo_data
 from tensorboard.plugins.profile import profile_plugin
 from tensorboard.plugins.profile import trace_events_pb2
+from shutil import copyfile
```

Comment:
Could you replace with `import shutil` and move it near to `import os`? Native imports come first, and python style is to import the module, then use it.

Reply:
I just have this 1 comment, and then lets merge. Thanks!

Reply:
Done. (Sorry, not sure how I missed this)